src/ontology/edits/oeo-physical.omn

```diff
@@ -6130,6 +6162,12 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/130",
     SubClassOf: 
         OEO_00000316
     
+    DisjointWith: 
```

Comment:
```suggestion
    DisjointWith: OEO_00030002
```

Reply:
What do you want exactly with this change? That I delete the term tracker item?